helm-values/vc-authn-oidc/dev.yaml

```diff
@@ -0,0 +1,94 @@
+ingressSuffix: -dev.apps.silver.devops.gov.bc.ca
+
+image:
+  repository: ghcr.io/bcgov/vc-authn-oidc
+  pullPolicy: IfNotPresent
+  tag: "2.0.0-beta1"
+
+acapyTenancyMode: single
+setNonRevoked: true
+useOobPresentProof: false
+useOobLocalDIDService: false
+controllerCameraRedirectUrl: wallet_howto
+controllerPresentationExpireTime: 300
+useHTTPS: true
+logLevel: INFO
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 512Mi
+  requests:
+    cpu: 10m
+    memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 2
+  targetCPUUtilizationPercentage: 80
+
+networkPolicy:
+  enabled: true
+  ingress:
+    enabled: true
+    namespaceSelector:
+      network.openshift.io/policy-group: ingress
+
+ingress:
+  enabled: true
+  annotations:
+   route.openshift.io/termination: edge
+
+acapy:
+  image:
+    repository: ghcr.io/hyperledger/aries-cloudagent-python
+    pullPolicy: IfNotPresent
+    tag: py3.9-0.9.0
+
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 3
+    targetCPUUtilizationPercentage: 80
+
+  argfile.yml:
+    admin-insecure-mode: false
+    auto-accept-invites: true
+    auto-accept-requests: true
+    auto-ping-connection: true
+    auto-provision: true
+    auto-respond-credential-offer: true
+    auto-respond-messages: true
+    auto-verify-presentation: true
+    genesis-transactions-list: /tmp/ledgers.yml
+    label: '{{ include "acapy.label" .}}'
+    log-level: info
+    monitor-ping: true
+    multitenant-admin: false
+    multitenant: false
+    public-invites: true
+    read-only-ledger: true
+    wallet-name: askar-wallet
+    wallet-storage-type: postgres_storage
+    wallet-type: askar
+    webhook-url: 'https://{{ include "vc-authn-oidc.host" . }}/webhooks'
+
+  ledgers.yml:
+    - id: SOVRINSandbox
+      is_production: true
+      is_write: true
+      genesis_url: 'https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis'
+    - id: BCovrinTest
+      is_production: true
+      genesis_url: 'http://test.bcovrin.vonx.io/genesis'
```

Comment:
Should this be BCovrin Dev? @WadeBarnes this matches the current configurations [here](https://github.com/bcgov/trust-over-ip-configurations/tree/main/openshift/templates/agent/config) but am wondering whether this is something we need to tweak.

Reply:
It was set as BCovrin Test since most wallets don't ready from BCovrin Dev.

Reply:
Ok, I am good keeping it as-is then.